### PR TITLE
Adds class for admin notes

### PR DESCRIPTION
**Pull request by thesuss**
Also fixes a bug I had created with actually adding notes. 

It looks the same, just made with .admin-notes instead of %pre

### CODE REVIEW

File: app/views/users/_member_payment_status.html.haml [hunk 1]
```diff
@@ -1,27 +1,28 @@
-.row.mt-2#userMemberStatus
-  - if defined?(error) && error
-    .alert.alert-danger= error
+#userMemberStatus
```
[the remark this answers is not in the document]
Comment (weedySeaDragon) [reply app/views/users/_member_payment_status.html.haml:1]: Should do this change as hound suggests

**Review by weedySeaDragon — CHANGES_REQUESTED** [review]
This change will be good.

Just need the 1 small change to `#userMemberStatus` -> `#user-member-status`

File: app/views/users/_member_payment_status.html.haml (hunk 1 above)
[the remark this answers is not in the document]
Comment (thesuss) [reply app/views/users/_member_payment_status.html.haml:1]: Its not mine. It has to do with stuff out of my control.

[the remark this answers is not in the document]
Comment (thesuss) [reply app/views/users/_member_payment_status.html.haml:1]: I think there is a reason it is named this way.

[the remark this answers is not in the document]
Comment (weedySeaDragon) [reply app/views/users/_member_payment_status.html.haml:1]: hm.  ok.

**Review by weedySeaDragon — APPROVED** [review]

**Review by patmbolger — APPROVED** [review]

**Pull request merged**